main/src/library/File.flix

```diff
@@ -15,6 +15,173 @@
  */
 namespace File {
 
+    ///
+    /// File types.
+    ///
+    pub enum FileType with Eq, Order, ToString {
```

Comment:
Should this be `public` or not?

Reply:
Yes, `public` is the right choice. That way users e.g. match on the result of the `fileType` function.